R/pkg/R/RDD.R

```diff
@@ -306,17 +306,26 @@ setMethod("checkpoint",
 #'\dontrun{
 #' sc <- sparkR.init()
 #' rdd <- parallelize(sc, 1:10, 2L)
-#' numPartitions(rdd)  # 2L
+#' getNumPartitions(rdd)  # 2L
 #'}
-#' @rdname numPartitions
+#' @rdname getNumPartitions
+#' @aliases getNumPartitions,RDD-method
+#' @noRd
+setMethod("getNumPartitions",
+          signature(x = "RDD"),
+          function(x) {
+            callJMethod(getJRDD(x), "getNumPartitions")
+          })
+
+#' numPartitions is the same as getNumPartitions
```

Comment:
Could you add a note that this is deprecated here ?

Also this is the title of the document, so its better to have this be something like `Gets the number of partitions of an RDD` cc @felixcheung